test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Rendering/HtmlHelperDisplayExtensionsTest.cs

```diff
@@ -94,6 +96,94 @@ public void Display_UsesTemplateName()
             viewEngine.Verify();
         }
 
+        public static TheoryData<FormatModel, string> DisplayFormatModels
+        {
+            get
+            {
+                return new TheoryData<FormatModel, string>
+                {
+                    {
+                        new FormatModel{ FormatProperty = Status.Created},
+                        "Value: CreatedKey"
+                    },
+                    {
+                        new FormatModel { FormatProperty = Status.Done},
+                        "Value: Done"
+                    }
+                };
+            }
+        }
+
+        public static TheoryData<FormatModel, string> DisplayUnformatedModels
```

Comment:
`EnumUnFormattedModels` and reuse this to create a similar test in `HtmlHelperEditorExtensionsTest`. (No need to dupe the theory data.)

Note "formatted" contains two t's (four times in this file).